rdr_service/genomic/genomic_job_components.py

```diff
@@ -1875,9 +1875,6 @@ def reconcile_metrics_to_data_files(self, genome_type, _gc_site_id):
             file_types_received = set([f.file_type for f in files])
             missing_data_files = required_files_set - file_types_received
 
-            if missing_data_files:
-                self.any_missing_data = True
-
```

Comment:
Should there be genomics changes in this PR?

Reply:
Clean up from proceeding PR, didn't want to open another PR for just this.